packages/ui/src/components/vuestic-components/va-input/VaInput.vue

```diff
@@ -130,6 +134,10 @@ const InputContextMixin = makeContextablePropsMixin({
     type: Number,
     default: 0,
   },
+  insidePagination: {
```

Comment:
This prop doesn't make a lot of sense for user. Can't we just style input from pagination component?

Reply:
Actually, do we need input component in pagination? Maybe just input is good enough? None of its advanced features are being used.

Reply:
done